docs/source/_static/data_processing_examples/starmap.py

```diff
@@ -0,0 +1,192 @@
+"""
+STARmap processing example
+==========================
+
+This notebook demonstrates the processing of STARmap data using starfish. The
+data we present here is a subset of the data used in this
+`publication <starmap>`_ and was generously provided to us by the authors.
+
+.. _starmap: https://doi.org/10.1126/science.aat5691
+
+Load example data
+-----------------
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import starfish
+import starfish.data
+from starfish.types import Axes
+from starfish.util.plot import (
+    diagnose_registration, imshow_plane, intensity_histogram
+)
+
+experiment = starfish.data.STARmap(use_test_data=True)
+stack = experiment['fov_000'].get_image('primary')
+
+###############################################################################
+# Visualize raw data
+# ------------------
+# In this starmap experiment, starfish exposes a test dataset containing a
+# single field of view. This dataset contains 672 images spanning 6 rounds
+# :code:`(r)`, 4 channels :code:`(ch)`, and 28 z-planes :code:`(z)`. Each image
+# is :code:`1024x1024 (y, x)`
+#
+# To examine this data, the vignette displays the max projection of channels and
+# rounds. Ideally, these should form fairly coherent spots, indicating that the
+# data are well registered. By contrast, if there are patterns whereby pairs of
+# spots are consistently present at small shifts, that can indicate systematic
+# registration offsets which should be corrected prior to analysis.
+
+ch_r_projection = stack.max_proj(Axes.CH, Axes.ROUND)
+
+f, ax = plt.subplots()
+imshow_plane(ch_r_projection, sel={Axes.ZPLANE: 15}, ax=ax)
```

Comment:
as a general style rule, for looking at these data, it's good practice to use cmap='gray' and plt.axis('off')

Reply:
i've noticed this a lot in the sphinx gallery examples in particular. this will look very strange to people who are accustomed to looking at image based transcriptomics data, but perhaps less strange to those who aren't accustomed to these data.